.github/workflows/autoupdate.yml

```diff
@@ -1,23 +1,23 @@
 name: Auto-update mamba
 on:
- schedule:
-   - cron: "0 */6 * * *"
+  schedule:
+    - cron: "0 */6 * * *"
 jobs:
   createPullRequest:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
```

Comment:
Please don't add comments. Dependabot doesn't update them and this they become stale

Reply:
I think it does? See this PR, for example: https://github.com/conda/conda/pull/13892

Reply:
Oh, that's new. We have a lot of these stale comments lingering in our internal repos 🤔